policykit/integrations/metagov/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from policyengine.models import PlatformAction, PlatformPolicy
+
+
+class ExternalProcess(models.Model):
+    json_data = models.CharField(max_length=500, blank=True, null=True)
+    policy = models.ForeignKey(PlatformPolicy, on_delete=models.CASCADE)
+    action = models.ForeignKey(PlatformAction, on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ['policy', 'action']
+
+# TODO add generic integrator models, so that Metagov can act as a connector for various external platforms
+# ExternalCommunity
+# ExternalUser
+# ExternalPlatformAction]
```

Comment:
Alright @amyxzhang I updated it to move all the metagov-specific views and models into  `integrations/metagov` –– not `clients/metagov` as I initially suggested. Metagov will also be behaving as an "integration" in the sense that it will act as a connector, creating PlatformActions for external platforms (Open Collective, Discourse, etc..). I think it makes sense to keep it all together under `integrations/metagov` rather than splitting out `integrations/metagov` and `libraries|clients/metagov`. These comments highlighted here show a preview of what I'm working on on another branch. I'll have some questions for you & Ricky regarding these generic "ExternalPlatformAction" and such, but I'll ask those in Slack 🙂 

@rickygv99 @amyxzhang  let me know what you think. This expands the meaning of PolicyKit "integrations" to also include "libraries" within them if desired.